Assets/MRTK/Services/InputSystem/MixedRealityInputSystem.cs

```diff
@@ -171,11 +174,13 @@ public override void Initialize()
             {
                 DebugUtilities.LogVerbose("MixedRealityInputModule added to main camera");
                 // There is no input module attached to the camera, add one.
-                CameraCache.Main.gameObject.AddComponent<MixedRealityInputModule>();
+                inputModule = CameraCache.Main.gameObject.AddComponent<MixedRealityInputModule>();
                 isInputModuleAdded = true;
             }
             else if ((inputModules.Length == 1) && (inputModules[0] is MixedRealityInputModule))
-            { /* Nothing to do, a MixedRealityInputModule was applied in the editor. */ }
+            {
+                inputModule = inputModules[0] as MixedRealityInputModule;
```

Comment:
What's the change here from before (i.e. why this used to be a no-op, but now is doing something?)

Reply:
The code here is saving the reference so that we don't need to look for the script later.